bucket/xnconvert.json

```diff
@@ -0,0 +1,49 @@
+{
+    "version": "1.90",
+    "description": "A fast and powerful batch image converter",
+    "homepage": "https://www.xnview.com/en/xnconvert/",
+    "license": "Freeware",
+    "suggest": {
+        "vcredist": "vcredist"
```

Comment:
So from the clear requirement of vcredist2019 you will force all of them to be installed?